### DISCUSSION TITLE
Cloning via fine-tuned yourTTS model on a new language.

### DISCUSSION THREAD

**Discussion by addyag93** [coqui-ai/TTS#2599] [General Q&A]
Hi,

I am able to fine-tune yourTTS model to a new language with optimal voice quality. But I wanted to use this model to clone the voice of native speaker in that language.
First question, is it possible to use this model to clone the voice.
Second: Assuming it is possible, I am trying to use `tts.tts_to_file(` but it is not able load my model correctly. I am not sure how to do it via "tts-cli" options.  

Can somebody help?

thanks
aditya

**Comment by Rahu16** [comment]
Hi @addyag93 I am also trying to build the pipeline yourTTS architecture to finetune it for a new language as the performance is better
But I was unable to find out enough info about it. Can you please provide some content document or code or anything which will help me to build this pipeline ?
